### DISCUSSION TITLE
可以实现自动打资源吗

### DISCUSSION THREAD

**Discussion by PUGENB** [xtyzhen/COC_helper_xtyzhen#4] [Q&A] [answered]
可以实现自动打资源吗
自动造兵搜索资源进攻

**Comment by xtyzhen** [comment, marked as answer]
已根据OCR以资源量为依据完成自动搜鱼（暂未开源），但自动进攻目前没有计划开发，这不仅破坏游戏平衡，而且容易被官方封禁。
如有相关需求，请使用其它脚本完成。
